typings/express-extension.d.ts

```diff
@@ -9,7 +9,7 @@ declare namespace Express {
 	export interface Request {
 		prefetchApiKey?: Resolvable<ApiKey>;
 
-		creds?: Creds;
+		creds?: import('../src').Creds;
```

Comment:
```suggestion
		creds?: import('../src/infra/auth/jwt-passport').Creds;
```
Just a nit to match the `ApiUser` import above. Personally I'd prefer imports at the top of the file instead of inline to make things more standard, but not necessary I guess.